packages/preview/src/browser/preview-link-normalizer.ts

```diff
@@ -21,10 +21,11 @@ import { MiniBrowserEndpoint } from '@theia/mini-browser/lib/browser/location-ma
 @injectable()
 export class PreviewLinkNormalizer {
 
+    protected urlScheme = new RegExp('^[a-z][a-z|0-9|\+|\-|\.]*:', 'i');
```

Comment:
Thank you for the quick fix. I wonder why there is a custom regex and not [URI.js](https://medialize.github.io/URI.js/) used.

(Refs https://stackoverflow.com/a/21462360/873282).